packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -20,6 +19,10 @@ import config from 'mwp-cli/src/config';
 import { API_PROXY_PLUGIN_NAME } from '../config';
 
 export const API_META_HEADER = 'X-Meta-Request-Headers';
+const MEMBER_COOKIE_NAME =
+	process.env.NODE_ENV === 'production' ? 'MEETUP_MEMBER' : 'MEETUP_MEMBER_DEV';
```

Comment:
would be nice if some of these were more consts but nice to have

Reply:
Yeah I had them as consts for awhile, but then needed a common place to import them from, and it made the package dependencies reeeeally messy because this name is needed in a few different packages that don't directly relate to each other. I have a Jira ticket to make a `mwp-config` package that would have zero MWP dependencies and therefore would be simple to import anywhere that needed 'shared' MWP config/const info, so I think this concern can be resolved there